os/Kconfig.devmng

```diff
@@ -0,0 +1,34 @@
+
```

Comment:
1. need to add belows.
```
#
# For a description of the syntax of this configuration file,
# see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
#
```
2. DM is in framework. Let's move this inside framework/src/dm.

Reply:
ok